### DISCUSSION TITLE
Sroll even when the cursor is hidden

### DISCUSSION THREAD

**Discussion by Schweber** [YaLTeR/niri#814] [Ideas]
Since https://github.com/YaLTeR/niri/pull/797 the cursor is unhidden on the first scroll wheel input and the window is only scrolled when another scroll wheel input with the unhidden cursor is given.

I would like the window to be scrolled immediately, also when the cursor is currently hidden. This is how `hyprland` behaves if i remember correctly (i'm not sure if it even unhides the cursor if you don't move it) and i find this to be more convenient. If this is a contentious viewpoint, then it could be made an option in the configuration.

**Comment by FreeFull** [comment]
This already has been fixed in niri v0.1.10.1

**Comment by Schweber** [comment]
```
niri -V                                                                                                                            
niri 0.1.10-1 (unknown commit)
```
I have that version but niri shows the behaviour that i describe. It first unhides the cursor and only scrolls when i give a second scroll wheel input with the unhidden cursor visible.

**Reply by YaLTeR** [reply]
Hmm, indeed. I'll have to double-check

**Reply by YaLTeR** [reply]
Actually, it seems to only happen on Firefox (GTK 3) and nautilus (GTK 4). My other clients (winit, Qt, Chromium) scroll just fine.

**Reply by Schweber** [reply]
RIght, i didn't think of checking in other apps, sorry. I'm experiencing it in Firefox as well, in kitty it works fine.
